src/Kunstmaan/NodeBundle/EventListener/NodeTranslationListener.php

```diff
@@ -46,22 +46,22 @@ class NodeTranslationListener
     /**
      * NodeTranslationListener constructor.
      *
-     * @param FlashBagInterface            $flashBag
+     * @param SessionInterface             $session
      * @param LoggerInterface              $logger
      * @param SlugifierInterface           $slugifier
      * @param RequestStack                 $requestStack
      * @param DomainConfigurationInterface $domainConfiguration
      * @param PagesConfiguration           $pagesConfiguration
      */
     public function __construct(
-        FlashBagInterface $flashBag,
+        SessionInterface $session,
```

Comment:
This should be changed according to my PR review comment. The change in the services.yml can be kept as is!